Allow use of last git commit message or a custom message (#94)
helen/action-wordpress-plugin-deploy: README.md

### Inputs
* `generate-zip` - Generate a ZIP file from the SVN `trunk` directory. Outputs a `zip-path` variable for use in further workflow steps. Defaults to false.

> * `commit-message` -  Use git last commit message if true, or "Update to version $VERSION from GitHub" if false else any text. Defaults to false.

Reviewer: I think we should rename this to `use-commit-message` to explicitly express the meaning of the flag. The current flag makes me think I should pass an actual message to this flag.

Reviewer: > if false else any text.

I think we should mention only `true` and `false` here to prevent any confusion. For example:

```suggestion
* `use-commit-message` -  Use git the latest git commit message as the svn commit message, otherwise use "Update to version $VERSION from GitHub". Defaults to false.
```

Author: Yes, this is better. Though some would still want to use resolved text. For example I currently use the below code to extract the version to supply to the github action 

```yml
    - name: Extract Version
      run:
        echo "STABLE_TAG=$(find . -iname README.TXT -exec grep -oiP -m 1 'stable\s+tag\s*:\s\K.*' {} \;)" >> $GITHUB_ENV
```
then
```yml
    - name: WordPress Plugin Deploy
      id: deploy
      uses: 10up/action-wordpress-plugin-deploy@stable
      env:
        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
        SLUG: ...
        VERSION: ${{ env.STABLE_TAG }}
        BUILD_DIR: build
```

Regarding clarity of what the input takes, a table would be able to convey the message better.

<table>
    <thead>
        <tr>
            <th>Input</th>
            <th>Description</th>
            <th>Default</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td ><code>use-commit-message</code></td>
            <td><span>Accepts either a boolean or text value</span>
<ul>
<li><code>true</code>&nbsp;:&nbsp;<span>Will use the last Git Commit message.</span></li>
<li><code>false</code>&nbsp;:&nbsp;<span>Will use "Update to version $VERSION from GitHub"</span></li>
<li><code>text</code>&nbsp;:&nbsp;<span>Any text value to be used as the commit message. Useful for when you want to dynamically generate the commit message.</span></li>
</ul>
</td>
 <td><code>false</code></td>
        </tr>
        <tr>
            <td><code>generate-zip</code></td>
            <td>...</td>
            <td>...</td>
        </tr>
    </tbody>
</table>

Or list items

#### Inputs

1. `use-commit-message` : Accepts either a boolean or text value. default `false`
   - `true` : Will use the last Git Commit message.
   - `false` : Will use "Update to version $VERSION from GitHub"
   - `text` : Any text value to be used as the commit message. Useful for when you want to dynamically generate the commit message.
 
2. `generate-zip` : ...

So if documented properly it can be made clear what inputs it takes, what it defaults to, etc. I was going to do a PR for the documentation but ran out of time.

You might also want to remove the extra git from `Use git the latest git...`
